### ISSUE TITLE
Some basic use questions

### ISSUE THREAD

**Issue by brucehohl** [andmarti1424/sc-im#131]
Just let me know if this is not the right /best place for such questions: 
1  How to get to top of help file (:h)?  Like in vim one would use "gg". 
2  How to get a file to autocalc when loaded.  Or how to autocalc after loading. 
3  How (where) to preserve ':set' options? 
4  The syntax for :fsum is seems to be ':int fsum'.  But the help file implies ':fsum'.    
5  If one selects, yanks then pastes a formula from column A to column B the references are not changed.  [A0+A1] does not become [B0+B1].     
6  A loaded file cannot be saved without reentering its path and file name.

**Comment by andmarti1424** [comment]
1. C-a moves you to bottom. Probably I should add that 'gg' shortcut as well.
2. if you set autocalc it should autocalc. if that does not happen.. may be we have a bug. please report it.
you can try :int rebuild_graph to see if that updates it.
3. you can store them in $HOME/.scimrc with commands such as:
SET DEBUG
SET NUMERIC..
4. there was a typo there. :sum was coded instead of :fsum. the correct command is :fsum
:int commands are more low level commands.. they are called internually.
5. Try the 'pC' command in normal mode, after yanking. that will update references. If you want to do this in just one cell, you can mark a cell with 'ma', and then paste it with 'ca'. if will update references as well.
6. this should not happen. I just tested this, and did not have this problem.ou 
i am saving it with just :w command. are you doing the same?

**Comment by brucehohl** [comment]
Thanks for that reply:
(1) I now see the navigation instructions at the beginning of the help file: Shift+G for bottom of file and Ctrl+a for top of file, etc.
(2) Recalc works correctly when I open sc-im from the command line.  My problem occurred when I opened the file from a Xfce/Thunar desktop launcher so no bug.  (I had no luck in creating a properly working Xfce/Thunar desktop launcher .... maybe a path problem )   
(3) I added ~/.scimrc and it works perfectly.
(4) Understood.
(5) Understood.  I re-read the help file and understand navigation & cell handling a bit better.
(6) Same as 2 above.
